vendor/knative.dev/reconciler-test/pkg/eventshub/resources.go

```diff
@@ -156,7 +156,7 @@ func Install(name string, options ...EventsHubOption) feature.StepFn {
 			// No event recording desired, just logging.
 			envs[EventLogsEnv] = "logger"
 			cfg["envs"] = envs
-			cfg["sink"] = sinkURL
```

Comment:
@pierDipi  this update comes in via 

https://github.com/knative-extensions/reconciler-test/commit/03cc77c11831ef010415c5bf6ecb38abf6b6780e